### ISSUE TITLE
[modules] What to do about module names

### ISSUE THREAD

**Issue by Clam-** [Clam-/pyBurlyBot#60]
Previously I was prefixing module names internally when loading them with `pyBurlyBot_` so that there wouldn't be any conflicts with other files/modules and imports.

This breaks multiprocessing on win32 because on win32 a new python process is created and then manually the desired module is attempted to be loaded by name from file (e.g. `pyBurlyBot_module.py`) which wouldn't exist because the actual file is `module.py`.

Do we just require module writers always use names that will most likely not conflict with anything else? (Somewhat difficult, consider moduleA called `photoinfo.py`, then also consider moduleB called `metadatasay.py` which uses a third party library called `photoinfo`. Because they share the same name, whichever is loaded first will be attempted to be used the second time the reference comes up.)
(Even less complicated is the module `wikipedia.py` which uses the `wikipedia` third party module.)

Or do we require all modules have a filename prefix such as "pbm_" or something?

Need decision.

**Comment by Clam-** [comment]
In the latter case such as we require modules to use a filename prefix, do we also require that prefixed is used in config file? Or do we allow non prefix name in config and auto magicking for convenience?

**Comment by Clam-** [comment]
So going from IRC convo, you are definitely wanting to force modules to have a prefixed module name? What kind of prefix? ```pyb_modulename``` ?
Glancing through https://pypi.python.org/pypi?%3Aaction=index I don't see anything starting with ```pyb_``` but I see many things starting with ```pyb```.

In saying this, are you currently modifying these things? Otherwise I've already rolled back the initial-internal prefixing of modules so my current thing will work locally, it would be easy enough to also add in the enforcement of module prefixing. y/n?

**Comment by gkistler** [comment]
Feel free to commit whenever.

I actually am sort of floundering on this particular issue.  It would be super helpful if you could provide some sort of test so I can replicate this issue and try various things.  If not feel free to roll with prefixed module names; all good.

**Comment by Clam-** [comment]
I don't know if this test works but something like this should easily show problem:
```python
#testmodule
from multiprocessing import Process

class TestProcess(Process):
	def run(self):
		while True:
			pass
		
def init(bot):
	TestProcess().start()
	return True
```

**Comment by gkistler** [comment]
Cool, ty.  Sticking that into a module called `clam_test.py` and enabling it throws this in startup:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\Python27\lib\multiprocessing\forking.py", line 347, in main
    self = load(from_parent)
  File "c:\Python27\lib\pickle.py", line 1378, in load
    return Unpickler(file).load()
  File "c:\Python27\lib\pickle.py", line 858, in load
    dispatch[key](self)
  File "c:\Python27\lib\pickle.py", line 1090, in load_global
    klass = self.find_class(module, name)
  File "c:\Python27\lib\pickle.py", line 1124, in find_class
    __import__(module)
ImportError: No module named pyBurlyBot_clam_test
```

**Comment by gkistler** [comment]
Addressed in 6e85a352542e4b5873f33f75028c2ba83c31a615
